src/descriptor/satisfied_constraints.rs

```diff
@@ -337,6 +337,34 @@ where
                     has_errored: false,
                 }
             }
+            &Descriptor::WshSortedMulti(ref smv) | &Descriptor::ShWshSortedMulti(ref smv) => {
+                SatisfiedConstraints {
+                    verify_sig: verify_sig,
+                    public_key: None,
+                    state: vec![NodeEvaluationState {
+                        node: &Any::from_smv_segwitv0(smv),
```

Comment:
Here we should first create a new miniscript as 
`let ms = &Miniscript::from_ast(smv.sorted_node()).expect("sorted node is also a valid ms");` and them call Any::from_segwitv0 and so on.

That should avoid editing the context.rs file

Reply:
As discussed on IRC, this does not work because of lifetime errors. I will think of a better way to do this, this can be left unaddressed.

Reply:
One way to approach this may be to always store the sorted vector in the `SortedMultiVec`.

Reply:
But then we lose the ability to even print out the original descriptor, right?

Reply:
We can't do that without `ToPublicKey` and is descriptor is `Descriptor<MiniscriptKey>`

Reply:
As discussed later, I will address this in a separate PR and we can leave this as `unimplemented!` for now

Reply:
Leaving this `unimplemented!`